src/main/java/org/sagebionetworks/web/client/widget/entity/browse/FilesBrowserViewImpl.java

```diff
@@ -60,13 +63,16 @@ public FilesBrowserViewImpl(FilesBrowserViewImplUiBinder binder, EntityTreeBrows
 			if (isEmpty) {
 				downloadHelp.setHelpMarkdown("There are no downloadable items in this folder.");
 				downloadHelp.setVisible(true);
```

Comment:
Is the the tooltip handling all of the cases that that the `downloadHelp` widget handled? If so, let's remove it.